### DISCUSSION TITLE
Change Field Names in Channel Layout settings

### DISCUSSION THREAD

**Discussion by ErhanGuneyKaradag** [ExpressionEngine/ExpressionEngine#2963] [Feature Requests]
Hi. I have an idea. 

There are fields that I use in common in different channels. For example like text_1 (text input) text_2 (text_input). I should be able to change the field name of that field in the layout page of that channel in whichever channel I am using.

**For example on the about us page.**

- field Name= About Us shortName= text_1 (text input)

on the contact page 

- field Name = E-mail shortName = still text_1.  (text input)

Thus, instead of seeing text_1 on the contact page, the field name will see E-mail.
I hope I was able to explain. 

Thank you.

**Comment by intoeetive** [comment]
Thank you for the feature request!

NB: this is related (but not identical) to https://github.com/ExpressionEngine/ExpressionEngine/issues/792
